cypress/e2e/applicant/apply.cy.js

```diff
@@ -0,0 +1,226 @@
+require("dotenv").config();
+
+const BASE_URL =
+  "https://dev-env.find-a-grant-support-dev.service.cabinetoffice.gov.uk/";
+
+const signInWithOneLogin = () => {
+  cy.contains("Sign in with GOV.UK One Login").click();
+
+  cy.origin("https://signin.integration.account.gov.uk", () => {
+    cy.contains("Sign in").click();
+    cy.get('[name="email"]').type(Cypress.env("oneLoginApplicantEmail"));
+    cy.contains("Continue").click();
+    cy.get('[name="password"]').type(Cypress.env("oneLoginApplicantPassword"));
+    cy.contains("Continue").click();
+    //cy.contains("Continue").click();
+  });
+};
+
+const saveAndContinue = () => {
+  cy.contains("Save and continue").click();
+};
+
+const yesSectionComplete = () => {
+  cy.get("[data-cy=cy-radioInput-option-YesIveCompletedThisSection]").click();
+  saveAndContinue();
+};
+
+const fillOutRequiredChecks = () => {
+  cy.contains("Required checks").click();
+
+  cy.contains("Enter the name of your organisation");
+  cy.get("[data-cy=cy-APPLICANT_ORG_NAME-text-input").type("My First Org");
+  saveAndContinue();
+
+  cy.contains("Choose your organisation type");
+  cy.get("[data-cy=cy-APPLICANT_TYPE-select]").should("have.value", "");
+  cy.get("[data-cy=cy-APPLICANT_TYPE-select]").select("Limited company");
+  cy.get("[data-cy=cy-APPLICANT_TYPE-select]").should(
+    "have.value",
+    "Limited company",
+  );
```

Comment:
do we need this check? we're validating the answers later on the review page anyway

Reply:
not necessarily but don't think it hurts

Reply:
Cypress recommends doing it this way